Source/Android/jni/ButtonManager.cpp

```diff
@@ -338,7 +338,7 @@ static void AddBind(const std::string& dev, sBind* bind)
 void Init(const std::string& gameId)
 {
   // Initialize our touchScreenKey buttons
-  for (int a = 0; a < 8; ++a)
+  for (int a = 0; a < 5; a = a + 4)
```

Comment:
You can use `a += 4`. A short comment explaining why you do 0 and 4 specifically would be nice as well.

Reply:
Updaded